lua/refactoring/treesitter/langs/go.lua

```diff
@@ -11,7 +11,8 @@ function Golang.new(bufnr, ft)
         version = Version:new(
             TreeSitter.version_flags.Scopes,
             TreeSitter.version_flags.Locals,
-            TreeSitter.version_flags.Classes
+            TreeSitter.version_flags.Classes,
```

Comment:
at some point I want to get rid of this now that we have the nodes.  I think the nodes should be enough to tell what type of operations we have.  But that is neither here nor there.

Reply:
I think we can start doing this right after the last scm's are moved into inline.